gr-uhd/grc/gen_uhd_usrp_blocks.py

```diff
@@ -114,7 +114,7 @@
 -   domain: message
     id: command
     optional: true
-    hide: ${'$'}{hide_cmd_port}
+    hide: ${'$'}{'all' if bool(eval(hide_cmd_port)) else 'none'}
```

Comment:
I'd greatly prefer that this change is in a different PR.

Reply:
This was a mistake. Good catch.